### DISCUSSION TITLE
Target with name 'Server=localhost\SQLEXPRESS;Database=MyMeetings;Trusted_Connection=True;' does not exist

### DISCUSSION THREAD

**Discussion by vikramvee** [kgrzybek/modular-monolith-with-ddd#305] [Q&A] [answered]
Getting this error while executing the build.ps1 script.

Target with name 'Server=localhost\SQLEXPRESS;Database=MyMeetings;Trusted_Connection=True;' does not exist. Available targets are:
  - ArchitectureTests
  - BuildAdministrationModuleIntegrationTests
  - BuildAndUnitTests
  - BuildMeetingsModuleIntegrationTests
  - BuildPaymentsModuleIntegrationTests
  - BuildSystemIntegrationTests
  - BuildUserAccessModuleIntegrationTests
  - Clean
  - Compile
  - CompileDbUpMigrator
  - CompileDbUpMigratorForIntegrationTests
  - CreateDatabase
  - MigrateDatabase
  - PrepareInputFiles
  - PrepareSqlServer
  - PrepareSUT
  - Restore
  - RunAdministrationModuleIntegrationTests
  - RunAllIntegrationTests
  - RunDatabaseMigrations
  - RunMeetingsModuleIntegrationTests
  - RunPaymentsModuleIntegrationTests
  - RunSystemIntegrationTests
  - RunUserAccessModuleIntegrationTests
  - UnitTests

**Comment by AlmarAubel** [comment]
What exact command are you using?

**Reply by vikramvee** [reply]
Sorry, missed to mention. Executing the below command.

`.\build MigrateDatabase "Server=localhost\SQLEXPRESS;Database=MyMeetings;Trusted_Connection=True;"`

**Reply by AlmarAubel** [reply]
Can you try this? 
`.\build MigrateDatabase --DatabaseConnectionString Server=localhost\SQLEXPRESS;Database=MyMeetings;Trusted_Connection=True;" `

**Reply by vikramvee** [reply, marked as answer]
Got below error after executing the command you mentioned:

 A connection was successfully established with the server, but then an error occurred during the login process. (provider: SSL Provider, error: 0 - The certificate chain was issued by an authority that is not trusted.

But resolved it by using the below command:

`.\build MigrateDatabase --DatabaseConnectionString "Server=localhost\SQLEXPRESS;Database=MyMeetings;Trusted_Connection=True;TrustServerCertificate=True"`

Looked like it executes fine untill I got another SQL exception:

> 
>  [19:37:27 INF] SQL exception has occurred in script: '1_0_0_0.0001_initial_structure.sql'
>  [19:37:27 ERR] Script block number: 1; Block line 2; Procedure ; Number 2714; Message: There is already an object named 'administration' in the database.
>    CREATE SCHEMA failed due to previous errors.
